daffodil-udf/src/test/java/org/badudfs/nonserializable/StringFunctions/Replace.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.badudfs.nonserializable.StringFunctions;
+
+import java.io.Serializable;
+
+import org.apache.daffodil.udf.FunctionClassInfo;
+
+@FunctionClassInfo(
+		name = "replace",
+		namespace = "com.ns.badudfs.StringFunctions"
```

Comment:
Can we use more XML namespace-y looking namespaces for all these examples? This look too much like java namespaces which might be confusing for people using these as examples? Maybe either our urn: style that we use for Daffodil or the common http:// style.

Reply:
I updated most of them to either use http://i.am.foo.com, urn:com-i-am-foo or com.i.am.foo style namespaces for variety's sake